src/backend/src/services/teams.services.ts

```diff
@@ -162,4 +162,60 @@ export default class TeamsService {
 
     return teamTransformer(updateTeam);
   }
+
+  /**
+   * Update the given teamId's team's leads
+   * @param submitter a user who's making this request
+   * @param teamId a id of team to be updated
+   * @param userIds a array of user Ids that replaces team's old leads
+   * @returns an updated team
+   * @throws if the team is not found, the submitter has no privilege, or any user from the given userIds does not exist
+   */
+  static async setTeamLeads(submitter: User, teamId: string, userIds: number[]): Promise<Team> {
+    const team = await prisma.team.findUnique({
+      where: { teamId },
+      ...teamQueryArgs
+    });
+
+    const teams = await prisma.team.findMany(teamQueryArgs);
```

Comment:
so i was refactoring my backend, and I think it makes sense to use our current service functions for these (aka use the getAllTeams service function instead of directly calling the prisma shit)